provider/datagen/src/tests/make_testdata.rs

```diff
@@ -18,33 +16,50 @@ use std::sync::Mutex;
 include!("../../tests/locales.rs.data");
 
 #[test]
-#[ignore] // has side effects, run manually
-fn generate_json_and_verify_postcard() {
-    simple_logger::SimpleLogger::new()
-        .env()
-        .with_level(log::LevelFilter::Info)
-        .init()
-        .unwrap();
-
-    let json_out = Box::new(
-        FilesystemExporter::try_new(Box::new(Json::pretty()), {
-            let mut options = ExporterOptions::default();
-            options.root = "tests/data/json".into();
-            options.overwrite = OverwriteOption::RemoveAndReplace;
-            options
+#[cfg(feature = "use_wasm")]
+fn make_testdata() {
+    // Only produce output if the variable is set. Test is hermetic otherwise.
+    let exporter: Box<dyn DataExporter> = if std::option_env!("TESTDATA_OUT").is_none() {
```

Comment:
Nit: `TESTDATA_OUT` looks like it takes a value of where you write the testdata, but it is only a boolean. How about `ICU4X_WRITE_TESTDATA`

Reply:
Sure.

I'm confused that you're leaving a nit comment but not giving approval. Either your comment is blocking, or it's a nit.

Reply:
Ok

Reply:
so it's not a nit?

Reply:
I changed my review from non-blocking to blocking